apps/opik-backend/src/main/java/com/comet/opik/infrastructure/db/TransactionTemplateImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.comet.opik.infrastructure.db;
+
+import io.r2dbc.spi.ConnectionFactory;
+import lombok.RequiredArgsConstructor;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+public class TransactionTemplateImpl implements TransactionTemplate {
```

Comment:
The implementation was not exposed to clients only use and depend on the interface

Reply:
Updated as discussed